Representable/representable
# Delete orgs

**Changes:**

- update models so that deleting org doesn't delete all communities submitted to it
- create view for deleting organization
- create URL for deleting organization
- add button for deleting organization and pop-up modal for confirming the action

**To test:**
- migrate
- run server
- create test org to delete
- create drive within that org
- submit community to that drive
- go to the dashboard -> manage the org to delete -> press red trash icon
- confirm that close and cancel work
- delete permanently 
- check that community still exists under "my communities" with drive=NULL and organization=NULL
- check that the org and the drive are both deleted from the db 

Closes #161
